### ISSUE TITLE
How can I see the qualitative results of test images?

### ISSUE THREAD

**Issue by chetanreddym** [davidtvs/Keras-LinkNet#1]
Thanks for sharing the code, David.
Is there a way where I can I save the test result images.

**Comment by davidtvs** [comment]
No, you'll have to implement it using [predict_generator](https://keras.io/models/sequential/#predict_generator) to get the predictions. Should be similar to the [test](https://github.com/davidtvs/Keras-LinkNet/blob/master/main.py#L111) function, pass it the test data generator and it'll return the predictions in a numpy array
